logic/src/commonMain/kotlin/com/wire/kalium/logic/data/e2ei/E2EIRepository.kt

```diff
@@ -345,24 +346,29 @@ class E2EIRepositoryImpl(
     override suspend fun fetchFederationCertificates() =
         discoveryUrl().flatMap {
             wrapApiRequest {
-                acmeApi.getACMEFederation(it)
+                acmeApi.getACMEFederationCertificateChain(it)
```

Comment:
will this work if the user is not federated ?

Reply:
As discussed over Wire: shouldn't work fine :D